src/printer.js

```diff
@@ -2094,7 +2094,11 @@ function printMethod(path, options, print) {
     group(
       concat([
         path.call(function(valuePath) {
-          return printFunctionParams(valuePath, print, options);
+          return comments.printComments(
```

Comment:
Babylon and Flow have different AST for this one.

Flow: http://astexplorer.net/#/gist/a7338343c53614ac8611f8599bfb5277/b7996c232ad7bfb8f911c3b326afe051edcdebdf

Babylon: http://astexplorer.net/#/gist/a7338343c53614ac8611f8599bfb5277/41170dbb3038f533743ac974ab2320e22d5ca718

In flow, if the comment is attached to FunctionExpression (which doesn't exist in the babylon ast) then it's not printed.